condDatasetSubmitter.py

```diff
@@ -214,14 +214,26 @@ def getCMSSWReleaseFromPath(thePath):
             return path
     raise ValueError('%s does not contain a slash-separated path to a CMSSW release. ERRROR.' % (thePath))
 
-def getDriverDetails(Type, B0T, HIon, pA, recoRelease):
+def getDriverDetails(Type, release, ds, B0T, HIon, pA, recoRelease):
+    str_era_hlt = 'Run2_2016'
+    print "AAA=======", release
+    if release.find("9_2_0")!= -1:
+        for ds_name in ds:
```

Comment:
we should be more inclusive about checking for 2017 releases
We'll surely have, at some point, 9_2_1, 9_2_13 etc

Logically, the check should be >= 9_2_X
where X can be anything